### DISCUSSION TITLE
在已有的Excel文件增加新的Sheet？

### DISCUSSION THREAD

**Discussion by bobli** [mini-software/MiniExcel#646] [General]
已经有一个Excel文件，需要往里面增加一个新的Sheet，并保存到原来的文件。
目前想到一个办法是把Excel文件所有Sheet的数据读出来，然后加上新Sheet的数据，再覆盖写入原来的文件。

**Comment by izanhzh** [comment]
我同样也有这个需求，并且我尝试进行了调整支持：https://github.com/izanhzh/MiniExcel/commit/f28f81811fff8ea3d63282d008bfb2c073295b87

目前遇到问题是单元格样式的问题，由于目前MiniExcel的style.xml是固定的，因此如果已有的Excel是由MiniExcel产生的，我的调整可以勉强满足需求，但是如果已有的Excel不是MiniExcel产生的，就会产生单元格样式错误问题

是否可以考虑封装MiniExcel对style.xml的操作能力，由于我对MiniExcel及Excel各个xml的很多细节还不熟悉，我暂时没有很好的办法解决

**Comment by izanhzh** [comment]
Resolved: #709
